pkg/move/technologies.go

```diff
@@ -0,0 +1,90 @@
+package move
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/afero"
+)
+
+type Manifest struct {
+	Technologies TechEntries `json:"technologies"`
+	Version      string      `json:"version"`
+}
+
+type TechEntries map[string]ArchEntries
+type ArchEntries map[string][]FileEntry
+
+type FileEntry struct {
+	Path    string `json:"path"`
+	Version string `json:"version"`
+	MD5     string `json:"md5"`
+}
+
+func copyByTechnology(fs afero.Afero) error {
+	logrus.Infof("Starting to copy (filtered) from %s to %s", sourceFolder, targetFolder)
+
+	filteredPaths, err := filterFilesByTechnology(fs, sourceFolder, strings.Split(technology, ","))
+	if err != nil {
+		return err
+	}
+
+	for _, path := range filteredPaths {
+		targetFile := filepath.Join(targetFolder, strings.Split(path, sourceFolder)[1])
+
+		err = fs.MkdirAll(filepath.Dir(targetFile), os.ModePerm) // check source dir's Stat.Mode
```

Comment:
maybe todo comment ?

Reply:
I removed the todo comment and added the stat mode check for sourcefolder and passed that one as mode, is this what you meant? I really forgot about it 😕 

https://github.com/Dynatrace/dynatrace-bootstrapper/pull/4/commits/a26ec38d7fa8e56ae0a5b200530ae2c4acd69a3a

https://github.com/Dynatrace/dynatrace-bootstrapper/pull/4/commits/9577a11ff5c1b156e3c6120d29790b43bfb91025